server/src/internalClusterTest/java/org/opensearch/cluster/coordination/AwarenessAttributeDecommissionIT.java

```diff
@@ -104,38 +104,52 @@ public void testDecommissionStatusUpdatePublishedToAllNodes() throws ExecutionEx
 
         logger.info("--> starting decommissioning nodes in zone {}", 'c');
         DecommissionAttribute decommissionAttribute = new DecommissionAttribute("zone", "c");
-        DecommissionRequest decommissionRequest = new DecommissionRequest(decommissionAttribute);
+        // Set the timeout to 0 to do immediate Decommission
+        DecommissionRequest decommissionRequest = new DecommissionRequest(decommissionAttribute, TimeValue.timeValueSeconds(0));
         DecommissionResponse decommissionResponse = client().execute(DecommissionAction.INSTANCE, decommissionRequest).get();
         assertTrue(decommissionResponse.isAcknowledged());
 
+        logger.info("--> Recieved decommissioning nodes in zone {}", 'c');
+        // Keep some delay for scheduler to invoke decommission flow
+        Thread.sleep(3000);
```

Comment:
nit: typo
3s sleep will slow down tests significantly can we try lower?

Reply:
Reduced to 100 milis. Tried few times. Can see it passing all the times. Looks safe.